Src/Couchbase.Linq.UnitTests/Proxies/IgnoreProxyFieldContractResolverTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+using Couchbase.Linq.Proxies;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using NUnit.Framework;
+
+namespace Couchbase.Linq.UnitTests.Proxies
+{
+    [TestFixture]
+    public class IgnoreProxyFieldContractResolverTests
```

Comment:
Jeff,

I think the IgnoreProxyFieldContractResolver and it's associated tests are leftovers, it doesn't appear to be referenced anywhere.